index.js

```diff
@@ -1,125 +1,201 @@
 'use strict'
 
 const nodePath = require('path')
-
-const commonExtname = require('common-extname')
-const commonPathPrefix = require('common-path-prefix')
-const glob = require('glob')
+const GlobSync = require('glob/sync').GlobSync
+const GLOBSTAR = require('minimatch').GLOBSTAR
+const hasMagic = require('glob').hasMagic
 const identifierfy = require('identifierfy')
-const flatten = require('lodash.flatten')
 
-function getPattern (path, source) {
-  const pattern = source.value.replace(/^glob:/, '').trim()
-  if (!pattern) {
-    throw path.buildCodeFrameError(`Missing glob pattern '${source.value}'`)
+const dirname = nodePath.dirname
+const relative = nodePath.relative
+const resolve = nodePath.resolve
+const fileSeparator = nodePath.sep
```

Comment:
I know `nodePath` is odd… perhaps `path` in the `ImportDeclaration` visitor should be renamed `astPath` or `babelPath`.